src/components/tests/settings-drawer.test.tsx

```diff
@@ -27,6 +28,7 @@ describe('Settings Drawer', () => {
         <SettingsDrawer settings={settings} onSettingsChange={onSettingsChange} open={true} />
       </SettingsContext.Provider>,
     );
+    expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
This is generating a massive snapshot file (+5000). I suggest we do something like this

``` ts
  it('should render correctly', () => {
    const wrapper = shallow(
      <SettingsContext.Provider value={settings}>
        <SettingsDrawer settings={settings} onSettingsChange={onSettingsChange} open={true} />
      </SettingsContext.Provider>,
    );
    expect(toJson(wrapper.dive())).toMatchSnapshot();
    wrapper.unmount();
  });

```
https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/dive.html

That will generate this:

```
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Drawer should render correctly 1`] = `
<WithStyles(ForwardRef(Drawer))
  PaperProps={
    Object {
      "className": "makeStyles-drawer-1",
    }
  }
  anchor="right"
  open={true}
>
  <WithStyles(ForwardRef(FormControl))
    component="fieldset"
  >
    <WithStyles(ForwardRef(FormLabel))
      className="makeStyles-legendLabel-2"
      component="legend"
    >
      Trajectory Animation
    </WithStyles(ForwardRef(FormLabel))>
    <WithStyles(ForwardRef(Divider)) />
    <ForwardRef(RadioGroup)
      className="makeStyles-trajGroup-3"
      onChange={[Function]}
      value={2}
    >
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="0"
        label="None"
        name="None"
        value={0}
      />
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="1"
        label="Fill"
        name="Fill"
        value={1}
      />
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="2"
        label="Follow"
        name="Follow"
        value={2}
      />
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="3"
        label="Outline"
        name="Outline"
        value={3}
      />
    </ForwardRef(RadioGroup)>
    <WithStyles(ForwardRef(FormLabel))
      className="makeStyles-legendLabel-2"
      component="legend"
    >
      Trajectory Diameter
    </WithStyles(ForwardRef(FormLabel))>
    <WithStyles(ForwardRef(Divider)) />
    <ForwardRef(RadioGroup)
      className="makeStyles-trajGroup-3"
      onChange={[Function]}
      value={1}
    >
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="0"
        label="Fix Size"
        name="Fix Size"
        value={0}
      />
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="1"
        label="Robot Size"
        name="Robot Size"
        value={1}
      />
    </ForwardRef(RadioGroup)>
    <WithStyles(ForwardRef(FormLabel))
      className="makeStyles-legendLabel-2"
      component="legend"
    >
      Trajectory Color
    </WithStyles(ForwardRef(FormLabel))>
    <WithStyles(ForwardRef(Divider)) />
    <ForwardRef(RadioGroup)
      className="makeStyles-trajGroup-3"
      onChange={[Function]}
      value={1}
    >
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="0"
        label="Theme"
        name="Theme"
        value={0}
      />
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="1"
        label="Robot Color"
        name="Robot Color"
        value={1}
      />
      <WithStyles(ForwardRef(FormControlLabel))
        className="makeStyles-flexBasis-4"
        control={<ForwardRef(WithStyles) />}
        key="2"
        label="Shades"
        name="Shades"
        value={2}
      />
    </ForwardRef(RadioGroup)>
  </WithStyles(ForwardRef(FormControl))>
</WithStyles(ForwardRef(Drawer))>
`;

```

With this method, we will be able to understand what's the component rendering by just looking at the snapshot.